README.md

```diff
@@ -2,6 +2,8 @@
 
 Eggplant is a music streaming service.
 
+![Eggplant](./screenshot.png)
```

Comment:
```suggestion
![Eggplant](https://user-images.githubusercontent.com/1935975/97783231-444af080-1b8e-11eb-8652-8eea2766d6d4.png)
```

Reply:
I recommend dropping the file from the repo and linking directly to a picture from this pull request.